src/main/resources/de/qabel/desktop/ui/accounting/login/login.fxml

```diff
@@ -5,82 +5,109 @@
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.*?>
-<VBox alignment="TOP_CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity"
-      prefHeight="672.0" prefWidth="369.0" styleClass="login" xmlns="http://javafx.com/javafx/8"
-      xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.accounting.login.LoginController">
+<VBox alignment="TOP_CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="672.0" prefWidth="369.0" styleClass="login" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.accounting.login.LoginController">
     <children>
-        <ImageView fitHeight="150.0" fitWidth="200.0" pickOnBounds="true" preserveRatio="true">
+        <ImageView fitHeight="203.0" fitWidth="139.0" pickOnBounds="true" preserveRatio="true">
             <image>
-                <Image url="@../../../../../../logo-invert.png"/>
+                <Image url="@../../../../../../logo-with-label-invert.png" />
             </image>
+         <VBox.margin>
+            <Insets />
+         </VBox.margin>
         </ImageView>
-        <VBox alignment="TOP_CENTER" prefHeight="445.0" prefWidth="342.0" styleClass="login-form">
+		<VBox alignment="TOP_CENTER" prefHeight="397.0" prefWidth="220.0" styleClass="login-form">
             <children>
                 <Label styleClass="login-title" text="%loginText">
                     <VBox.margin>
-                        <Insets bottom="10.0" top="10.0"/>
+                        <Insets bottom="10.0" top="10.0" />
                     </VBox.margin>
                 </Label>
-                <ChoiceBox fx:id="providerChoices"/>
-                <TextField fx:id="user" alignment="CENTER" promptText="%loginBoxAccountUserName" styleClass="user"
-                           text="testuser">
+				<ChoiceBox fx:id="providerChoices">
+					<VBox.margin>
+						<Insets left="10.0" right="10.0"/>
+					</VBox.margin>
+				</ChoiceBox>
+				<TextField fx:id="user" alignment="CENTER" prefHeight="26.0" prefWidth="220.0"
+						   promptText="%loginBoxAccountUserName" styleClass="user" text="testuser">
                     <VBox.margin>
-                        <Insets/>
+						<Insets left="10.0" right="10.0"/>
                     </VBox.margin>
                 </TextField>
 
-                <TextField fx:id="email" alignment="CENTER" managed="false" promptText="%loginUserEMail" styleClass="user">
+                <TextField fx:id="email" alignment="CENTER" managed="false" prefHeight="26.0" promptText="%loginUserEMail" styleClass="user">
```

Comment:
I assume that EVERY input element needs the height of 26 so we have to add this to every field or it will be inconsistent. This can be automated by using css (`-fx-pref-height`). If possible, a padding sounds more reasonable to me than prefHeight.